Autopilot/AttitudeManager/UART2.c

```diff
@@ -61,22 +66,14 @@ void InitUART2()
 
 //	IPC7 = 0x4400;	// Mid Range Interrupt Priority level, no urgent reason
 
-        IPC7 = 0x4400;	// Mid Range Interrupt Priority level, no urgent reason
+  IPC7 = 0x4400;	// Mid Range Interrupt Priority level, no urgent reason
 
 	IFS1bits.U2TXIF = 0;	// Clear the Transmit Interrupt Flag
 	IEC1bits.U2TXIE = 0;	// Enable Transmit Interrupts
 	IFS1bits.U2RXIF = 0;	// Clear the Recieve Interrupt Flag
 
-#if PATH_MANAGER && !GPS_OLD
-	IEC1bits.U2RXIE = 1;	// Enable Recieve Interrupts
-#else
-        IEC1bits.U2RXIE = 0;	// Enable Recieve Interrupts
```

Comment:
Double check that this doesn't break anything. The else statement could be troublesome. 

If I'm not mistaken UART 2 is used for the debug cable....So we don't need a receive interrupt. Double check to be sure.